collector/index.md

```diff
@@ -0,0 +1,425 @@
+---
+title: 'pganalyze Collector'
+backlink_href: /docs
+backlink_title: 'Documentation'
+---
+
+The pganalyze collector periodically queries your database and sends metrics and metadata (as "snapshots") to
+the pganalyze app. This is the only component that talks directly to your database. For details on setting up
+the collector, see the [collector installation guide](/docs/install/02_installing_the_collector). The collector
+is open-source and the code is available [on GitHub](github.com/pganalyze/collector).
+
+
+# Configuration Settings
+
+The collector can be configured either through a `INI` config file (in a normal install) or environment variables
+(typically for running via its Docker image). Most settings can be configured through either mechanism (if both
+are present, the config file takes precedence). Note that a single collector instance can monitor more than one
+database server (though this is not supported when configuring through environment variables). The `[pganalyze]`
+section describes settings that apply to all servers, and the other sections are named after the database server
+they correspond to, and define settings for just that server. When using environment variables, the setting applies
```

Comment:
`sections are named after the database server they correspond to` => we should clarify that its a name you chose (doesn't have to correlate with the server name, or hostname - in fact the example we usually give is `server1` and `server2`)